Source/WebCore/platform/CaretAnimator.h

```diff
@@ -36,6 +36,11 @@ enum class CaretAnimatorType : uint8_t {
     Alternate
 };
 
+enum class CaretAnimatorStopReason : uint8_t {
+    Default,
+    CaretRectChanged,
+};
```

Comment:
Slightly strange to see this before the forward declarations. Would you mind moving these enums below line 50?